decidim-censuses/app/jobs/decidim/censuses/remove_duplicates_job.rb

```diff
@@ -0,0 +1,20 @@
+module Decidim
+  module Censuses
+    class RemoveDuplicatesJob < ApplicationJob
+
+      queue_as :default
+
+      def perform
+        duplicated = Census.select(:id_document)
+                           .group(:id_document)
+                           .having('count(id_document)>1')
+                           .map(&:id_document)
+
+        duplicated.each do |id_doc|
```

Comment:
Instead of each, prefer find_each. Find each is less performant for small groups of objects but is needed if you can retrieve many objects.